types/react-beautiful-dnd/index.d.ts

```diff
@@ -28,6 +28,10 @@ export interface DragStart {
     source: DraggableLocation;
 }
 
+export interface DragUpdate extends DragStart {
+    destination?: DraggableLocation;
```

Comment:
This should be `destination?: DraggableLocation | null;`, [as the Flow type is](https://github.com/atlassian/react-beautiful-dnd/blame/master/src/types.js#L261):

```
destination: ?DraggableLocation,
```

In Flow, ? is [the maybe type](https://flow.org/en/docs/types/maybe/), which can be undefined or null.